holoviews/plotting/mpl/element.py

```diff
@@ -497,8 +497,10 @@ class ColorbarPlot(ElementPlot):
     colorbar = param.Boolean(default=False, doc="""
         Whether to draw a colorbar.""")
 
-    cbar_width = param.Number(default=0.05, doc="""
-        Width of the colorbar as a fraction of the main plot""")
+    clipping_colors = param.Dict(default={'NaN': ('w', 1)}, doc="""
```

Comment:
I understand what I expect to happen to NaN values but an example of how to set the min and max colors would be nice (in the docstring).